webapp/certification/views.py

```diff
@@ -12,4 +12,44 @@
 
 
 def certification_home():
-    return flask.render_template("certification/index.html")
+
+    # Desktop section
+    desktop_releases = []
+    desktop_vendors = []
+
+    for release in api.certified_releases(limit="0")["objects"]:
```

Comment:
I think the certifiedreleases endpoint also supports queries like `?desktops__gte=1&laptops__gte=1`